docs/installation-and-operations/installation/docker/README.md

```diff
@@ -70,6 +70,16 @@ your `docker-compose.yml` file is stored (`compose` in this case).
 If you want to start from scratch and remove the exsiting data you will have to remove these volumes via
 `docker volume rm compose_opdata compose_pgdata`.
 
+### Configuration
+
+In the compose folder you will find the file `docker-compose.yml` which can be edited. Please be aware that only those variables shall be edited which are documented as not everything is meant to be configured or bend.
```

Comment:
@machisuji is that actually the way to go, or would you recommend using a `docker-compose.override.yml` for configuration?

Reply:
waiting for @machisuji

Reply:
@adam-op It's best to use a `docker-compose.override.yml` as @wielinde suggested so users can just download the latest `docker-compose.yml` from the deploy repo without having to worry about losing their own changes.

Reply:
okay, I will rewrite the docs with using the mentioned **docker-compose.override.yml**

Reply:
I did the requested changes, please could you approve/merge? Thank you